deeplinkdispatch-processor/src/main/java/com/airbnb/deeplinkdispatch/DeepLinkProcessor.java

```diff
@@ -313,6 +313,15 @@ private void generateDeepLinkDelegate(String packageName) throws IOException {
         .addStatement("throw new $T($S)", AssertionError.class, "No instances.")
         .build();
 
+    MethodSpec supportsUri = MethodSpec.methodBuilder("supportsUri")
+        .addModifiers(Modifier.PUBLIC, Modifier.STATIC)
+        .returns(TypeName.BOOLEAN)
+        .addParameter(String.class, "uriString")
```

Comment:
Wasn't sure if it was better to take a `String` or a `Uri`. I don't really feel strongly either way, it'd be just as easy to call `uri.toString()` before passing it into `parseUri()`.